go.mod

```diff
@@ -1,12 +1,24 @@
 module github.com/gravitational/trace
 
-go 1.15
+go 1.17
```

Comment:
Do you mean to upgrade the version here?

Reply:
Just realized teleport is on 1.17 as well, probably good to upgrade here actually

Reply:
going to revert to 1.15 as `api` hasn't been bumped to 1.17 yet